### PR TITLE
Morfi

### PR DESCRIPTION
**Pull request by gWhy-j**

### CODE REVIEW

**Comment by llamatester** [comment]
The MorFi adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs MorFi
    
    🦙 Running MORFI adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 03 Nov 2024 00:00:00 GMT
    End Date:	Mon, 04 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MORPH 👇
    Backfill start time: 29/10/2024
    NO METHODOLOGY SPECIFIED
    End timestamp: 1730678400 (2024-11-04T00:00:00.000Z)
    Block: 302.47 k
    Total volume: 50.22 k
    Daily volume: 2.23 k

**Comment by llamatester** [comment]
The MorFi adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs MorFi
    
    🦙 Running MORFI adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 03 Nov 2024 00:00:00 GMT
    End Date:	Mon, 04 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    error fetching block TypeError: Cannot read properties of null (reading 'getBlock')
        at _getCurrentChainBlock (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:84:59)
        at getCurrentChainBlock (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:76:22)
        at _lookupBlock (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:204:44)
        at /home/runner/work/dimension-adapters/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:155:26
        at /home/runner/work/dimension-adapters/dimension-adapters/node_modules/p-limit/index.js:23:31
        at run (/home/runner/work/dimension-adapters/dimension-adapters/node_modules/p-limit/index.js:23:43)
        at /home/runner/work/dimension-adapters/dimension-adapters/node_modules/p-limit/index.js:45:20
    MORPH 👇
    Backfill start time: 29/10/2024
    NO METHODOLOGY SPECIFIED
    End timestamp: 1730678400 (2024-11-04T00:00:00.000Z)
    Block: 302.46 k
    Total volume: 50.22 k
    Daily volume: 395
    Daily fees: 4
    Total fees: 449

**Comment by llamatester** [comment]
The MorFi adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs MorFi
    
    🦙 Running MORFI adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 03 Nov 2024 00:00:00 GMT
    End Date:	Mon, 04 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MORPH 👇
    Backfill start time: 29/10/2024
    NO METHODOLOGY SPECIFIED
    End timestamp: 1730678400 (2024-11-04T00:00:00.000Z)
    Block: 302.47 k
    Total volume: 50.22 k
    Daily volume: 395
    Daily fees: 4
    Total fees: 449

File: package-lock.json
Comment (g1nt0ki) [inline package-lock.json:1]: please revert change to the lock file

File: helpers/chains.ts [hunk 1]
```diff
@@ -203,26 +204,4 @@ const DOGE = "doge";
...
-  MANTLE,
-  MOONRIVER,
-  OKEXCHAIN,
-  OPTIMISM,
-  POLYGON,
-  RONIN,
-  XDAI,
-  ZKSYNC,
-};
+export { ARBITRUM, AURORA, AVAX, BITCOIN, BOBA, BSC, CELO, DOGE, ETHEREUM, FANTOM, HARMONY, HECO, LITECOIN, MANTLE, MOONRIVER, OKEXCHAIN, OPTIMISM, POLYGON, RONIN, XDAI, ZKSYNC };
```
Comment (g1nt0ki) [inline helpers/chains.ts]: also, revert this

File: package-lock.json
Comment (dylanghj) [reply package-lock.json:1]: @g1nt0ki  but, using the Morph network requires a library update.

**Comment by llamatester** [comment]
The MorFi adapter exports:
        
 
 
    > adapters@1.0.0 test
    > ts-node --transpile-only cli/testAdapter.ts dexs MorFi
    
    🦙 Running MORFI adapter 🦙
    ---------------------------------------------------
    Start Date:	Sun, 03 Nov 2024 00:00:00 GMT
    End Date:	Mon, 04 Nov 2024 00:00:00 GMT
    ---------------------------------------------------
    
    MORPH 👇
    Backfill start time: 29/10/2024
    NO METHODOLOGY SPECIFIED
    End timestamp: 1730678400 (2024-11-04T00:00:00.000Z)
    Block: 302.47 k
    Total volume: 50.22 k
    Daily volume: 395
    Daily fees: 4
    Total fees: 449

File: helpers/chains.ts (hunk 1 above)
Comment (dylanghj) [reply helpers/chains.ts]: fixed

File: package-lock.json
Comment (dylanghj) [reply package-lock.json:1]: image

Comment (dylanghj) [reply package-lock.json:1]: `npm test` with original package-lock.json

```
error fetching block TypeError: Cannot read properties of null (reading 'getBlock')
    at _getCurrentChainBlock (/Users/gh/who-is-dylan/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:84:59)
    at getCurrentChainBlock (/Users/gh/who-is-dylan/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:76:22)
    at _lookupBlock (/Users/gh/who-is-dylan/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:204:44)
    at /Users/gh/who-is-dylan/dimension-adapters/node_modules/@defillama/sdk/build/util/blocks.js:155:26
    at /Users/gh/who-is-dylan/dimension-adapters/node_modules/p-limit/index.js:23:31
    at run (/Users/gh/who-is-dylan/dimension-adapters/node_modules/p-limit/index.js:23:43)
    at /Users/gh/who-is-dylan/dimension-adapters/node_modules/p-limit/index.js:45:20
```

**Pull request closed without merging**